changelog_unreleased/cli/10274.md

```diff
@@ -0,0 +1,12 @@
+#### Add `--no-plugin-search` to turn off plugin autoloading (#10274 by @fisker)
+
+To turn off plugin autoloading, use `{ pluginSearchDirs: false }` in JavaScript code or `--no-plugin-search` when using Prettier CLI. You can also add `{ pluginSearchDirs: false }` to your config file.
```

Comment:
What does ``use `{ pluginSearchDirs: false }` in JavaScript code`` mean?

Reply:
I guess it does the same as `--no-plugin-search`.

Reply:
Oh I meant the "in JavaScript code" part. Because later it says to add that to the config file, so I didn't understand what "JavaScript code" is this

Reply:
Oh IC IC 😅 Maybe it refers to using Prettier as a JS library, not via CLI?

Reply:
Aaahhhh that makes sense! Maybe something like:

> To turn off plugin autoloading, use `--no-plugin-search` when using Prettier CLI or add `{ pluginSearchDirs: false }` to options in `prettier.format()` or to the config file.